service/src/main/resources/changelog/changesets/20220823_run.yaml

```diff
@@ -0,0 +1,41 @@
+databaseChangeLog:
+  - changeSet:
+      id: "1"
+      author: chrisl
+      changes:
+        - createTable:
+            tableName: run
+            columns:
+              - column:
+                  name: id
+                  type: uuid
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: engine_id
+                  type: varchar(255)
+                  constraints:
+                    nullable: true
+              - column:
+                  name: run_set_id
+                  type: uuid
+                  constraints:
+                    nullable: false
+                    foreignKeyName: fk_run_run_set
+                    references: run_set(id)
+              - column:
+                  name: entity_id
+                  type: varchar(255)
+                  constraints:
+                    nullable: false
```

Comment:
Should this be nullable instead as we want to allow workflow submission without entity inputs?